detox/src/devices/drivers/AndroidDriver.js

```diff
@@ -30,6 +29,7 @@ class AndroidDriver extends DeviceDriverBase {
 
     this.invocationManager = new InvocationManager(this.client);
     this.matchers = new AndroidExpect(this.invocationManager);
+    this.UiDevice = new UiDeviceProxy(this.invocationManager).getUIDevice();
```

Comment:
What about init'ing uidevice lazily? I'd rather not have this fragile code triggered in projects where it is never used... makes sense?

Reply:
I take it back, leave as-is